src/fmu/dataio/_model/global_configuration.py

```diff
@@ -137,6 +137,36 @@ def __iter__(self) -> Any:
     def __getitem__(self, item: str) -> StratigraphyElement:
         return self.root[item]
 
+    def _get_index(self, name: str) -> int:
+        """
+        Get the index of a stratigraphic element by name.
+        """
+        return list(self.root.keys()).index(name)
+
+    def get_official_name(self, name: str) -> str:
+        """
+        Get the official SMDA name of a stratigraphic element.
+        name: name of stratigraphic element
+        """
+        if name in self.root:
+            return self.root[name].name
+        return ""
```

Comment:
SInce you have implemented the `__getitem__` previously we could actually drop the root  🙂 
```suggestion
        if name in self:
            return self[name].name
        return ""
```

Reply:
👍